docs/development/configuration.md

```diff
@@ -66,6 +66,6 @@ If you add configuration options to your `package.json` then these will override
 
 ## Configuration Options
 
-Please see [https://docs.renovatebot.com/configuration-options/](https://docs.renovatebot.com/configuration-options/) for a list of user-facing configuration options.
+Please read [our list of user-facing configuration options](../usage/configuration-options.md).
```

Comment:
those docs links should probably use the live docs, as the repo files are missing a lot generated content

Reply:
```suggestion
Please read [our list of user-facing configuration options](https://docs.renovatebot.com/configuration-options/).
```